packages/fxa-settings/src/components/Settings/PageRecoveryPhoneSetup/index.tsx

```diff
@@ -82,7 +93,17 @@ export const PageRecoveryPhoneSetup = (_: RouteComponentProps) => {
     // one code can be valid at the same time if the user clicks “resend code” to
     // account for SMS transmission delay. (This will change in FXA-11039)
     // try/catch is in the component that calls this function
-    await account.addRecoveryPhone(phoneData.phoneNumber);
+    try {
+      await account.addRecoveryPhoneWithJwt(phoneData.phoneNumber);
+    } catch (e) {
+      if (e && e.code === 401 && e.errno === 110) {
```

Comment:
We should probably pull these into constants at some point.

Reply:
Good call. Which reminds, @nshirley add this nice little helper function in this PR. I'll update my checks accordingly. https://github.com/mozilla/fxa/pull/19468/files#r2356422594